apisix/cli/ngx_tpl.lua

```diff
@@ -592,7 +592,7 @@ http {
             set $dubbo_method                '';
             {% end %}
 
-            access_by_lua_block {
+            rewrite_by_lua_block {
```

Comment:
I am afraid we'd better have a discussion about this, this change breaks the original design purposes.

Reply:
how about this change

Reply:
APISIX does not currently mount the `rewrite_by_lua_block` phase of openresty.